gnat2goto/driver/ureal_to_binary.adb

```diff
@@ -151,7 +145,9 @@ package body Ureal_To_Binary is
                                          Exponent_Bias : Positive)
                                          return String is
       Is_Negative : constant Boolean := Number < Ureal_0;
-      Int_Part : String (1 .. 2**(Exponent_Bits - 1));
+      --  this buffer needs to be big enough to store the integer part
+      --  which is one bit bigger than 2^exp_bits
+      Int_Part : String (1 .. 2**(Exponent_Bits + 1));
```

Comment:
A better comment description here might be helpful - the current comment superficially looks to be inconsistent with the code (it's not - it just takes a bit of thinking about to realise this...). A better comment might be something like `...which is one bit wider than the exponent`

Reply:
It's not one bit wider than the exponent though, an 8 bit exponent would have a 512 bit `Int_Part` according to this

Reply:
Done.

Reply:
Ah right - so this surely means then that the comment and the code don't match? The code is not `(2^exp_bits)+1` it's `2^(exp_bits+1)` ?

Reply:
Yes,  I think the comment is still wrong

Reply:
Maybe fixed.